Makefile

```diff
@@ -152,65 +152,70 @@ test-deps-gpu: test-deps
 
 # help: build-tests                    - build all tests (C, C++, Fortran)
 .PHONY: build-tests
+build-tests: test-deps
```

Comment:
I think I'm actually walking back this suggestion that I had made. I think building the tests are useful enough (without running them) that maybe we just add the dependency of the CPP tests on `catch` to this list?

Reply:
Skipping per discussion